config/crd/bases/dbaas.redhat.com_mongodbatlasconnections.yaml

```diff
@@ -0,0 +1,130 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: mongodbatlasconnections.dbaas.redhat.com
+spec:
+  group: dbaas.redhat.com
+  names:
+    kind: MongoDBAtlasConnection
+    listKind: MongoDBAtlasConnectionList
+    plural: mongodbatlasconnections
```

Comment:
Maybe it is obvious, but do the plural and singular elements allow camel case or kebab-case? The value `mongodbatlasconnection` (and plural) can be hard to read.

Reply:
I think this is conventional and they are generated by Operator SDK anyway :)